backend/spec/requests/memos_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Memos' do
+  describe 'MemoAPI' do
+    context 'タイトルとコンテンツが有効な場合' do
+      let(:valid_memo_params) { attributes_for(:memo) }
```

Comment:
IMO(In My Opinion)

以下のように書き換えると、リクエストボディがどのような構造になるか分かりやすくて個人的に好みです！
（修正するかどうかは任せます！）

```suggestion
      let(:valid_memo_params) { { title: Faker::Lorem.sentence(word_count: 3), content: Faker::Lorem.paragraph(sentence_count: 5) } }
```

Reply:
確かに、書き手としてはattributes_forが楽ですが、どのような値が入るのか誰が見てもわかるという点ではそちらの方が良さそうだと僕自身も思いました！修正します！